src/__tests__/http-test.js

```diff
@@ -29,6 +29,14 @@ import {
 
 import { graphqlHTTP } from '../index';
 
+type Server = () => {|
```

Comment:
Leave a TODO to improve typing after we switch to TS and will have typings for `supertest` and other libs.